doc/reference/cardano-node-cli-reference.md

```diff
@@ -98,5 +99,45 @@ The `genesis` command contains the following sub commands:
 * `hash`: retrieves the hash value
 
 *cardano-cli text-view*
-The `text-view` command contains the following sub command:
-* `decode-cbor`: prints a text view file, as decoded CBOR.
+The `text-view` command contains the following subcommand:
+* `decode-cbor`: prints a text view file as decoded CBOR.
+
+
+
+## Advanced Commands
+
+`kes-period-info`: This command runs the following checks on your operational certificate and your operational certificate issue counter:
+- Do the counters of the issue counter and operational certificate match?
+- Do the counters match what is currently in the node state?
+- Does the KES key period specified in your operational certificate fall within the current KES key period?
+We essentially check the predicates of the OCERT rule in the cardano-ledger specification, and we print additional diagnostic information as follows:
+```
+✓ The counters match what is in the node's protocol state
+✓ Operational certificate's kes period is within the correct KES period interval
```

Comment:
Not sure why the `kes` above is uncapitalised whilst the `KES` below is capitalised.